packages/plugins/src/foundry/get-bundle-info.ts

```diff
@@ -35,6 +37,19 @@ const broadcasting = args[2] === 'true'
     const { artifactFolder, buildInfoFolder, canonicalConfigFolder } =
       await getPaths()
 
+    const execAsync = promisify(exec)
+    const forgeConfigOutput = await execAsync('forge config --json')
```

Comment:
We already call `await execAsync('forge config --json')` once in `getPaths()` above. It's not the slowest operation, but it's not instant (~0.1s), so it'd be great if we could refactor this to only call `execAsync` once. Maybe we could pass in the foundryConfig as a parameter to `getPaths()`